### ISSUE TITLE
What happend to subtitles 

### ISSUE THREAD

**Issue by plants247** [yuliskov/SmartTube#3667]
Before I updated today there was a whole list of languages i can select for subtitles to be automatically translated to, but after updating Smartube that list has shorted. Are There any plans to bring the original sized list or no?.  Thank you so much for reading and have a good day or night.

**Comment by pmeHA** [comment]
Yes, the list of languages ​​is very limited. It would be nice to get back to normal...

**Comment by roorq** [comment]
I have the same , no full list of translations, request to restore.
